test/validate/fbp_ferc1_test.py

```diff
@@ -34,20 +34,3 @@ def test_vs_bounds(pudl_out_ferc1, live_dbs, cases):
 
     for case in cases:
         pv.vs_bounds(fbp_ferc1, **case)
-
-
-def test_self_vs_historical(pudl_out_ferc1, live_dbs):
-    """Validate fuel by plants vs.
-
-    historical data.
-    """
-    if not live_dbs:
-        pytest.skip("Data validation only works with a live PUDL DB.")
-    else:
-        fbp_ferc1 = pudl_out_ferc1.fbp_ferc1()
-        for f in ["gas", "oil", "coal", "waste", "nuclear"]:
-            fbp_ferc1[f"{f}_cost_per_mmbtu"] = (
-                fbp_ferc1[f"{f}_fraction_cost"] * fbp_ferc1["fuel_cost"]
-            ) / (fbp_ferc1[f"{f}_fraction_mmbtu"] * fbp_ferc1["fuel_mmbtu"])
```

Comment:
I wonder if this calculation is why we have unweighted quantile checks for the FBP table?

Reply:
Kinda? we're covering that using a validation model: 
https://github.com/catalyst-cooperative/pudl/blob/71e9059f256c7e96766a2bffcd3a070630e774bc/dbt/models/ferc1/out_ferc1__yearly_steam_plants_fuel_by_plant_sched402/schema.yml#L44

the model does the cost per mmbtu calculation, so the quantile check is unweighted

Reply:
It just seemed weird and possibly wrong to me (even though I'm the one that made the old test) that it would *not* be weighted, but seeing that the cost per MMBTU is being constructed from these already aggregated values, rather than a bunch of individual records with very different weights, it seemed like maybe there was no other choice.

Reply:
sounds plausible!